site/en/blog/richer-pwa-installation/index.md

```diff
@@ -0,0 +1,185 @@
+date: 2021-04-23
+authors:
+  - mustafa
+tags:
+  - progressive-web-apps
+hero: "image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/H5CevJUnOWSewM02r4on.jpg"
+alt: "New install surface for progressive web apps"
+---
+
+{% Aside 'caution' %}
+This is an experimental UI, and could potentially change in the future depending on developer, user, 
+and partner feedback. We are also planning on expanding the UI to consider use cases where users need to be 
+educated about powerful features like WebSockets. 
+{% endAside %}
+
+
+## Introduction {: #introduction }
+
+Mobile devices and the introduction of device vendor app stores have changed users' mental model
+of how to discover, evaluate and install software. Users are now so familiar with app stores, and
+the additional information that is provided through app stores such as context about the app,
+social feedback, ratings etc that you see the app store metaphor emerging in Desktop operating
+systems including ChromeOS, Mac and Windows. 
+
+ 
+## Challenge with today's install surfaces {: #today }
+
+Today, if a user wants to install a PWA, an infobar and modal overlay appears with minimal
+information. If they continue to install, the process is over too quickly without giving
+context to the user. This goes against their expectations of installing apps and can leave them
+somewhat confused about what has happened.  
+
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/VwB1V3K61vQMs1htKJpY.png", 
+alt="An example of PWA install UI.", width="360", height="632" %}
+
+To enable developers to provide installed experiences on par with native experiences
+Chrome is introducing a new install surface, Richer Install, that allows developers to add a
+description and screenshots to their manifest file and have it appear in a bottomsheet dialog
+within Chrome for Android. 
+
+This gives developers the opportunity to create a more enticing install process
+that better aligns to user expectations and that mimics their existing mental model
+of installed experiences. 
+
+{% Columns %}
+{% Column %}
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/SpStAtUk8Zp5iwi9yqKP.jpg", 
+alt="Richer Install UI Expanded", width="342", height="722" %}
+{% Aside %}
+Richer Install UI Expanded
+{% endAside %}
+{% endColumn %}
+{% Column %}
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/k7r4yKqrh6iOm2XyZHfw.jpg", 
+alt="Richer Install UI Collapsed", width="342", height="722" %}
+{% Aside %}
+Richer Install UI Collapsed
+{% endAside %}
+{% endColumn %}
+{% endColumns %}
+
+## Backwards compatibility {: #compatibility }
+
+Websites that do not include at least one screenshot to their manifest file will continue to receive 
+the existing prompts. This may change in the future depending on uptake of the developer community 
+and users' reaction.  
+
+## Previewing the UI {: #previewing }
+
+This UI works in Chrome Dev & Chrome Canary on Android, with the flag `#mobile-pwa-install-use-bottom-sheet` enabled in `chrome://flags`.
```

Comment:
This is what I was commenting on at the top. Please use specific version numbers.